ksqldb-tools/src/test/java/io/confluent/ksql/tools/migrations/MigrationsTest.java

```diff
@@ -266,6 +266,7 @@ private void shouldApplyMigrations() throws Exception {
     assertThat(applyStatus, is(0));
 
     verifyMigrationsApplied();
+    assertThat(MIGRATIONS_CLI.parse("--config-file", configFilePath, "apply", "-v", "2").runCommand(), is(1));
```

Comment:
Can we add a comment to explain why this check is here? Really it should be its own test but duplicating the rest of this test just for the sake of testing this feels a bit overkill, so I'm fine with leaving it in this test but we should be explicit about what it's testing / why it's here.